custom/ewsghana/tasks.py

```diff
@@ -189,6 +190,13 @@ def migrate_email_settings(domain):
     endpoint = GhanaEndpoint.from_config(config)
     migrate_email = EmailSettingsSync(domain)
 
+    for user in iter_docs(WebUser.get_db(), WebUser.ids_by_domain(domain)):
+        for notification in ReportNotification.by_domain_and_owner(domain, user['_id'], stale=False):
+            notification.delete()
```

Comment:
@gcapalbo You're right that's bad solution so I think I'm not able to fix email reports on ews-ghana domain since all reports are badly migrated and we can't distinct migrated reports from newly created.